src/app/home-page/home-logged-out/home.component.html

```diff
@@ -320,8 +320,10 @@ <h5 class="mb-0">
           <span>Discourse forum about helpful Dockstore topics.</span>
         </div>
       </div>
-      <div fxFlex.md="33" fxFlex.gt-md="33" fxFlex.lt-md="100">
-        <div class="my-3 twitter" #twitter></div>
+      <div class="mt-timeline">
+        <div id="mt-body" class="mt-body" role="feed">
```

Comment:
Hmm, I think you can remove these two divs since you moved them into the mastodon component